src/_locales/en_US/messages.json

```diff
@@ -20,6 +20,12 @@
     "disabled_for_these_domains": {
         "message": "Privacy Badger is disabled for the domains listed below. You can add a domain either via the Privacy Badger icon in the address bar or by typing the domain (not a URL) in the box and clicking the \"Add domain\" button."
     },
+    "dnt_noaction": {
+        "message": "No tracking for "
+    },
+    "dnt_tooltip": {
+        "message": "This domain promises to not track you."
```

Comment:
I think this way is more readable. Also, having periods at the end of full sentences looks more professional, so I'd prefer to keep it.